src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html

```diff
@@ -219,6 +169,55 @@ <h3 class="margin-top margin-top-sm">No Olap Tables have been defined</h3>
+            </div>
+            <div class="olap-tables-list margin-bottom">
+                <!-- OLAP table edit area -->
+                <div class="panel" data-bind="visible: showEditOlapTableArea()">
+                    <form autocomplete="off">
+                        <div class="padding padding-sm flex-horizontal item" data-bind="with: editedOlapTableSandbox()">
+                            <div class="flex-grow" data-bind="validationElement: tableName">
+                                <input type="text" class="form-control input-sm" placeholder="Table name"
+                                       data-bind="textInput: tableName" />
+                            </div>
+                            <div class="flex-grow" data-bind="validationElement: documentIdColumn">
+                                <input type="text" class="form-control input-sm" placeholder="Enter ID column name (override the default '_id')"
+                                       data-bind="textInput: documentIdColumn" />
+                            </div>
+                            <div class="flex-noshrink">
+                                <button type="button" class="btn btn-success btn-sm" title="Add this ID column name" data-bind="click: $root.saveEditedOlapTable"><i class="icon-tick"></i></button>
+                                <button type="button" class="btn btn-default btn-sm" title="Cancel" data-bind="click: $root.cancelEditedOlapTable"><i class="icon-cancel"></i></button>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+                <!-- The olap tables list -->
+                <div class="text-center text-muted" data-bind="if: editedOlapEtl().olapTables().length === 0 && !showEditOlapTableArea()">
+                    <i class="icon-lg icon-empty-set"></i>
+                    <h3 class="margin-top margin-top-sm">No settings have been defined.</h3>
+                    <h3 class="margin-top margin-top-sm">The default ID column name is <code>_id</code>. Click Add Settings to override this name per table.</h3>
+                </div>
+                <div data-bind="foreach: editedOlapEtl().olapTables">
+                    <div class="panel panel-hover">
+                        <div class="padding padding-sm flex-horizontal item">
+                            <div class="row flex-grow">
+                                <div class="col-sm-4">
+                                    <div class="widthlabel"><label class="superscript">Table</label><strong data-bind="text: tableName"></strong><span class="text-warning" data-bind="visible: dirtyFlag().isDirty">&nbsp;*</span></div>
+                                </div>
+                                <div class="col-sm-4">
+                                    <div class="widthlabel"><label class="superscript">Id Column Name</label><strong data-bind="text: documentIdColumn"></strong></div>
```

Comment:
sometimes you use ID vs Id. Let's unify.

Reply:
done